backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -52,30 +55,31 @@ fun BpkTheme(
 ) {
   val typography = BpkTypography(defaultFontFamily = fontFamily)
   val colors = if (isSystemInDarkTheme()) BpkColors.dark() else BpkColors.light()
+  val shapes = BpkShapes()
 
-  CompositionLocalProvider(
-    LocalBpkTypography provides typography,
-    LocalBpkColors provides colors,
-    LocalContentColor provides colors.textPrimary,
-    LocalElevationOverlay provides null,
+  MaterialTheme(
+    typography = typography.toMaterialTypography(fontFamily),
```

Comment:
do we still need the typography/colour/shape matching? in our components we always set these etc and as material is no longer exposed the mapping shouldn't be necessary anymore (or at least very minimal)

Reply:
Having the full compatibility might set expectations that using material is supported by us, however we do not - minimal support would work in our favour here. It's relatively easy for us to test if any of the mapping is required internally - just remove it and see if any snapshots change. 
It'll be harder to remove the mapping the longer we keep it